admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -173,6 +202,30 @@ export const ApplicationPage: React.FunctionComponent = () => {
                                         <>
                                         </>
                                     </Modal>
+                                    <React.Fragment>
+                                        <Modal variant={ ModalVariant.small } titleIconVariant="warning" isOpen={ showDeleteModal }
+                                            onClose={ () => setShowDeleteModal(false) }
+                                            title={ `Permanently delete ${ eventType.name }` }>
+                                            { <b>{ eventType.name }</b> } {`from  ${ bundle ? bundle.display_name :
+                                                <Spinner /> }/${ (applicationTypesQuery.loading
+                                             || applicationTypesQuery.payload?.status !== 200) ?
+                                                <Spinner /> : applicationTypesQuery.payload.value.displayName } will be deleted. 
+                                                If an application is currently sending this event, it will no longer be processed.`}
+                                            <br />
+                                            <br />
+                                            Type <b>{ eventType.name }</b> to confirm:
+                                            <br />
+                                            <TextInput type='text' onChange={ handleDeleteChange }
+                                                value={ eventType.name } name="name" id='name' isRequired />
```

Comment:
this is no longer working after being set to value vs. defaultValue... not sure why?

Reply:
Ah... 

What is happening here is that we don't update the `eventType.name` ever on the delete modal. And that's fine.
It used to work, because `defaultValue` only used the first value, ignores the changes and instead it keeps a record on its own.
When `value` is set, it uses that value always.

See: https://en.reactjs.org/docs/uncontrolled-components.html#default-values

I suggest to remove `value` too so that it uses its own internal state.

Reply:
okay, I was just reading that doc snippet again from when you posted it prior - thanks for explaining!